python/lsst/daf/butler/core/_butlerUri/_butlerUri.py

```diff
@@ -308,6 +308,16 @@ def geturl(self) -> str:
         """
         return self._uri.geturl()
 
+    def getrooturl(self) -> str:
+        """Return the base root URI in string form.
+
+        Returns
+        -------
+        url : `str`
+            String form of root URI.
+        """
+        return self._uri.scheme + "://" + self._uri.netloc
```

Comment:
To be consistent with the other APIs this should return a new ButlerURI and not a string.